firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallOptions.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.functions;
+
+import java.util.concurrent.TimeUnit;
+import okhttp3.OkHttpClient;
+
+/** An internal class for keeping track of options applied to an HttpsCallableReference. */
+class HttpsCallOptions {
```

Comment:
Question: Are there some functions that could take more time?
Should this be configurable/part of public API?

Reply:
NVM. I see it down there :)